Back out "[AOTInductor] Include constants in AOTInductor .so file. (#108473)"

bertmaher opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #109243

Original commit changeset: 9494d031e3ac

Original Phabricator Diff: D49075977

Differential Revision: [D49243049](https://our.internmc.facebook.com/intern/diff/D49243049/)

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

bertmaher committed:
Back out "[AOTInductor] Include constants in AOTInductor .so file. (#108473)"

Original commit changeset: 9494d031e3ac

Original Phabricator Diff: D49075977

Differential Revision: [D49243049](https://our.internmc.facebook.com/intern/diff/D49243049/)

[ghstack-poisoned]

mikekgfb reviewed (approved):
The PR reverted by the present PR breaks CPU aot inductor

bertmaher committed:
Update on "Back out "[AOTInductor] Include constants in AOTInductor .so file. (#108473)""


Original commit changeset: 9494d031e3ac

Original Phabricator Diff: D49075977

Differential Revision: [D49243049](https://our.internmc.facebook.com/intern/diff/D49243049/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 ipiszy ngimel yf225 chenyang78 kadeng muchulee8 aakhundov

[ghstack-poisoned]

bertmaher commented:
We can't actually back this out, because the diff is important for a *different* production model.  Argh.  Fix forward, we shall.
